README.md

```diff
@@ -74,37 +74,8 @@ Widget counterAppBody(BuildContext context, WidgetHandle use) {
     ),
   );
 }
-
-// Or, until static metaprogramming roles around:
-class CounterAppBody extends RearchConsumer {
-  const CounterAppBody({super.key});
-
-  @override
-  Widget build(BuildContext context, WidgetHandle use) {
-    final (count, incrementCount) = use(countManager);
-    final countPlusOne = use(countPlusOneCapsule);
-    return Scaffold(
-      appBar: AppBar(title: Text('ReArch Demo')),
-      floatingActionButton: FloatingActionButton(
-        onPressed: incrementCount,
-        tooltip: 'Increment',
-        child: Icon(Icons.add),
-      ),
-      body: Center(
-        child: Text(
-          '$count + 1 = $countPlusOne',
-          style: TextTheme.of(context).headlineLarge,
-        ),
-      ),
-    );
-  }
-}
 ```
 
-*Note: the `@rearchWidget` above requires
```

Comment:
TODO: may need to update the README with `RearchWidget()` assuming `@rearchWidget` won't work